static/app/components/charts/groupStatusChart.tsx

```diff
@@ -0,0 +1,150 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import MarkLine from 'sentry/components/charts/components/markLine';
+import MiniBarChart from 'sentry/components/charts/miniBarChart';
+import {LazyRender} from 'sentry/components/lazyRender';
+import {t} from 'sentry/locale';
+import type {Group, TimeseriesValue} from 'sentry/types';
+import type {Series} from 'sentry/types/echarts';
+import {formatAbbreviatedNumber} from 'sentry/utils/formatters';
+import theme from 'sentry/utils/theme';
+
+function asChartPoint(point: [number, number]): {name: number | string; value: number} {
+  return {
+    name: point[0] * 1000,
+    value: point[1],
+  };
+}
+
+const EMPTY_STATS: ReadonlyArray<TimeseriesValue> = [];
+
+type Props = {
+  data: Group;
+  statsPeriod: string;
+  groupStatus?: string;
+  height?: number;
+  showMarkLine?: boolean;
+  showSecondaryPoints?: boolean;
+};
+
+function GroupStatusChart({
```

Comment:
This new component is largely the same as the pre-existing `<GroupChart/>` component, but I would still prefer to have work contained in this new file to avoid any unintended changes to the existing component.